Clossing.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+enum Type { Shirt, Jeans, Jacket };
+string eType[] = { "Shirt", "Jeans", "Jacket" };
+
+class Clothing {
+
+public:
+
+    string name;
```

Comment:
set Clothing data to private: 
    string name;
    string description;
    string location;
    string color;
    int size;
    Type type;